jitbuilder/control/Jit.cpp

```diff
@@ -120,12 +123,14 @@ initializeJitBuilder(TR_RuntimeHelper *helperIDs, void **helperAddresses, int32_
    // Create a bootstrap raw allocator.
    //
    TR::RawAllocator rawAllocator;
+   TR::BackingMemoryAllocator backingAllocator(rawAllocator, 1<< 20);
```

Comment:
They don't *have* to be linked together. That's just how they're used currently....I don't believe there's a requirement for the `BackingMemoryAllocator` to die just because the `SegmentAllocator` dies...